<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisciplinesDialog</class>
 <widget class="QDialog" name="DisciplinesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item>
        <widget class="QLineEdit" name="nameLine">
         <property name="placeholderText">
          <string>Название предмета</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dateLine">
         <property name="placeholderText">
          <string>Дата посещения</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="scoreLine">
         <property name="placeholderText">
          <string>Оценка</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
